tests/wasm/simple.rs

```diff
@@ -62,6 +64,20 @@ pub unsafe fn simple_raw_pointers_work(a: *mut u32, b: *const u8) -> *const u32
     a
 }
 
+#[wasm_bindgen]
+pub unsafe fn simple_option_raw_pointers_work(
+    a: Option<*mut u32>,
+    b: Option<*const u8>,
+) -> Option<*const u32> {
+    *a.unwrap() = *b.unwrap() as u32;
+    a.map(|ptr| ptr as *const _)
+}
+
+#[wasm_bindgen]
+pub unsafe fn simple_option_nonnull_work(a: Option<NonNull<u32>>) -> Option<NonNull<u32>> {
```

Comment:
```suggestion
pub fn simple_option_nonnull_work(a: Option<NonNull<u32>>) -> Option<NonNull<u32>> {
```

Manipulating raw pointers isn't actually unsafe, just dereferencing them.